app/controllers/items_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ItemsController < ApplicationController
+
+  def edit
+    @item = Item.find(params[:id])
+  end
+
+  def show
+    @item = Item.find(params[:id])
+  end
+  
+  def update
+    @item = Item.find(params[:id])
+    if @item.update(item_params)
+      flash[:alert] = "Item successfully updated!"
+      redirect_to "/items/#{@item.id}"
+    else
+      render 'edit'
+    end
+  end
+  
+  private
+  
+  def item_params
+    params.permit(:name, :description, :unit_price, :status)
+  end
+end
```

Comment:
This `ItemsController` setup looks great and it is very similar to the way I set up my `AdminMerchantsController` `update` method.